src/contracts/atlas/Escrow.sol

```diff
@@ -554,8 +554,8 @@ abstract contract Escrow is AtlETH {
         // Update solverTracker with returned data
         solverTracker = abi.decode(data, (SolverTracker));
 
-        // Verify that the solver repaid their borrowed solverOp.value
-        (, success) = _validateBalances();
+        // Verify that the solver repaid their borrowed solverOp.value by calling `reconcile()`
+        (,, success) = _solverLockData();
```

Comment:
If possible we should add back in the check from _validateBalances() in here so that we can leave room for the postSolverHook to contribute more funds

Reply:
Good catch, added back the check to allow possibility of postSolver to cover anything the solver didn't, here https://github.com/FastLane-Labs/atlas/pull/225/commits/7da60831a731ff95eb02fca065c50a07f50c445c

Reply:
Updated it to also check `calledBack` here https://github.com/FastLane-Labs/atlas/pull/225/commits/8af6f6c24548a7d577b0fcad63c28de284341995